src/Forex/ForexData.cs

```diff
@@ -0,0 +1,147 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using Newtonsoft.Json;
+
+using Azure.Migrate.Export.Common;
+using Azure.Migrate.Export.HttpRequestHelper;
+using Azure.Migrate.Export.Models;
+
+namespace Azure.Migrate.Export.Forex
+{
+    public static class ForexData
+    {
+        private static Dictionary<string, double> ExchangeRatesUSD;
+        private static double ExchangeRate = 1.0;
+        private static ExchangeRateStates ExchangeRateState = ExchangeRateStates.Unfetched;
+
+        public static Dictionary<string, double> GetExchangeRatesUSD()
+        {
+            return ExchangeRatesUSD;
+        }
+
+        public static ExchangeRateStates GetExchangeRateState()
+        {
+            return ExchangeRateState;
+        }
+
+        public static double GetExchangeRate()
+        {
+            return ExchangeRate;
+        }
+
+        public static void UpdateExchangeRatesWithUSDFallback()
+        {
+            List<KeyValuePair<string, string>> currencies = InitializationData.GetSupportedCurrenciesInitializationData();
+            ExchangeRatesUSD = new Dictionary<string, double>();
+
+            foreach(var kvp in currencies)
+                if (!ExchangeRatesUSD.ContainsKey(kvp.Key))
+                    ExchangeRatesUSD.Add(kvp.Key, 1.0);

```

Comment:
This leads to incorrect data used as workaround. Let's avoid this.

Reply:
Yup, thanks for the great suggestion, removed the population of false data in dictionary.